release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1576,6 +1584,11 @@ The `authentication` and `openshift-apiserver` Operators now ignore the `oauth-a
 +
 For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1948553[*BZ#1948553*]. 
 
+* Previously, following an etcd process, the next process would not start until the relevant ports were released. With the addition of SO_REUSEADDR to this process, the ports can be reused immediately. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1927942[BZ#1927942].
```

Comment:
This bug's a duplicate, so do we need the RN?
```suggestion
* Previously, following an etcd process, the next process did not start until the relevant ports were released. With the addition of `SO_REUSEADDR` to this process, the ports can be reused immediately. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1927942[BZ#1927942].
```

Packages and functions are usually [styled](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) with backticks.